### PR TITLE
Calendar management (Task No. 36)

### PR DESCRIPTION
**Pull request by netzarbeiter**

### CODE REVIEW

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 1]
```diff
@@ -1,52 +1,59 @@
 ---
-title: 'Schedule management'
-description: 'The event management is a separate module in the backend called "Events", which you can find in the group "Contents".'
+title: 'Calendar management'
+description: 'The calendar management is a separate module in the backend called "Events", which you can find in the
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md]:
```suggestion
description: 'The calendar management is a separate module in the back end called "Events", which you can find in the 
```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 2]
```diff
@@ -1,52 +1,59 @@
...
     - /en/core-extensions/calendar/calendar-management/
 weight: 10
 ---
 
-{{% notice warning %}}
-This article is machine translated.
-{{% /notice %}}
-
-The event management is a separate module in the backend called "Events", which you can find in the group "Contents". There you can create several calendars, which contain the individual appointments or events. By using several calendars, a categorization of the entries is possible.
+The event management is a separate module in the backend called "Events", which you can find in the "Content" group.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md]:
```suggestion
The event management is a separate module in the back end called "Events", which you can find in the "Content" group. 
```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 3]
```diff
@@ -1,52 +1,59 @@
...
-Just like content elements, calendars can also be protected. The events of the calendar are then only shown to registered members.
+Just like content elements, calendars can also be protected. The events of the calendar are then only shown to 
+registered members.
 
-**Protect archive:** Here you activate the access protection.
+**Protect calendar:** Here you activate the access protection.
 
-**Allowed member groups:** Here you define which member groups should have access to the calendar after logging in to the frontend.
+**Allowed member groups:** Here you define which member groups should have access to the calendar after logging in to 
+the frontend.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md]:
```suggestion
the front end.
```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 4]
```diff
@@ -1,52 +1,59 @@
...
-**Sorting:** Here you define the order of the comments.
+**Sort order:** Here you define the order of the comments.
 
-**Comments per page:** Here you can set the number of comments per page. Contao automatically creates a page break when it is needed.
+**Comments per page:** Here you can set the number of comments per page. Contao automatically creates a page break when 
+it is needed.
 
-**Moderate comments:** If you select this option, comments will not appear on the website immediately, but will appear after you have shared them in the backend.
+**Moderate comments:** If you select this option, comments will not appear on the website immediately, but will appear 
+after you have shared them in the backend.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md]:
```suggestion
after you have shared them in the back end.
```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 5]
```diff
@@ -1,52 +1,59 @@
...
 
-**Allow BBCode:** If you select this option, your visitors can use [BBCode](https://de.wikipedia.org/wiki/BBCode) to format their comments. The following tags are supported:
+**Allow BBCode:** If you select this option, your visitors can use [BBCode](https://de.wikipedia.org/wiki/BBCode) to
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md]:
```suggestion
**Allow BBCode:** If you select this option, your visitors can use [BBCode](https://en.wikipedia.org/wiki/BBCode) to 
```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 6]
```diff
@@ -85,78 +98,110 @@ The URL is:
...
 **Start time:** Here you enter the start time of the event.
 
 **End time:** Here you enter the end time of the event. To create an event with an open end, do not fill out this field.
 
 **Start date**: Here you enter the start date of the event.
 
-**End date:** Here you enter the end date of the event. If you do not fill in this field, Contao automatically assumes that the event will last one day.
+**End date:** Here you enter the end date of the event. If you do not fill in this field, Contao automatically assumes 
+that the event will last one day.
+
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:187]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 7]
```diff
@@ -85,78 +98,110 @@ The URL is:
...
+### Redirect target
+
+The redirection destination determines to which page a visitor is redirected when clicking on an event, usually the page 
+on which the frontend module "Eventleser" is integrated to display the complete event.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md]:
```suggestion
on which the front end module "Event reader" is integrated to display the complete event.
```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 8]
```diff
@@ -85,78 +98,110 @@ The URL is:
...
+**Article:** Here you select the destination article.
+
+**Link target:** Here you enter the URL of the external target page.
+
+**Open in a new window:** Here you can determine whether the external target page is opened in a new browser window or 
+not.
+
+| Redirect target | Explanation |
+| ---------------------- | ----------- |
+| Use default | You will be redirected to the page you specified in the archive settings. On this page the frontend module "news reader" should be included. |
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md]:
```suggestion
| Use default | You will be redirected to the page you specified in the archive settings. On this page the front end module "event reader" should be included. |
```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 9]
```diff
@@ -202,79 +249,73 @@ To create a new event, click on the desired archive ![Edit Calendar](/de/icons/e
...
 
-**Repetitions:** If you enter a value greater than 0 here, the event will not be displayed after the specified number of repetitions.
+**Recurrences:** If you enter a value greater than 0 here, the event will not be displayed after the specified number 
+of repetitions.
 
-### Annexes
+### Enclosures
 
-Enclosures, also called "Enclosures" in the context of RSS feeds, are files that are linked to an event; these files are both exported in the RSS feed and offered for download on the website.
+Enclosures, also called "Enclosures" in the context of RSS feeds, are files that are linked to an event; these files
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:295]:
```suggestion
In the context of RSS feeds, enclosures are files that are linked to an event; these files 
```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 10]
```diff
@@ -202,79 +249,73 @@ To create a new event, click on the desired archive ![Edit Calendar](/de/icons/e
...
 
 **CSS class:** Here you can add a CSS class to the event.
 
 **Disable comments:** Here you deactivate the comment function for an event.
 
-### Publication {#publication}
+### Publish settings
 
-As long as an event is not published, it is not displayed in the frontend. You already know this behaviour from pages and articles. In addition to the manual publication you have the possibility to activate events automatically at a certain date.
+As long as an event is not published, it is not displayed in the frontend. You already know this behaviour from pages
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md]:
```suggestion
As long as an event is not published, it is not displayed in the front end. You already know this behaviour from pages 
```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 11]
```diff
@@ -202,79 +249,73 @@ To create a new event, click on the desired archive ![Edit Calendar](/de/icons/e
...
+As long as an event is not published, it is not displayed in the frontend. You already know this behaviour from pages 
+and articles. In addition to the manual publication you have the possibility to activate events automatically at a 
+certain date.
 
 **Publish event:** Here you can publish the event.
 
-**Display from:** Here you activate the event on a specific date.
+**Show from:** Here you activate the event on a specific date.
 
 **Show until:** Here you deactivate the event on a certain date.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:319]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 12]
```diff
@@ -202,79 +249,73 @@ To create a new event, click on the desired archive ![Edit Calendar](/de/icons/e
...
-| Page | The redirection is to a specific page in the page structure. |
-| Article | The redirection is to a specific article. |
-| Individual URL | The redirection is to an individual URL. |
+**Enclosures:** Here you select the files you want to link to the event.
 
 ### Expert Settings
 
 **CSS class:** Here you can add a CSS class to the event.
 
 **Disable comments:** Here you deactivate the comment function for an event.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:307]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 13]
```diff
@@ -202,79 +249,73 @@ To create a new event, click on the desired archive ![Edit Calendar](/de/icons/e
...
-
-**Open in a new window:** Here you can determine whether the external target page is opened in a new browser window or not.
-
-| Forwarding destination | Explanation |
-| ---------------------- | ----------- |
-| Standard | You will be redirected to the page you specified in the archive settings. On this page the frontend module "news reader" should be included. |
-| Page | The redirection is to a specific page in the page structure. |
-| Article | The redirection is to a specific article. |
-| Individual URL | The redirection is to an individual URL. |
+**Enclosures:** Here you select the files you want to link to the event.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:301]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 14]
```diff
@@ -202,79 +249,73 @@ To create a new event, click on the desired archive ![Edit Calendar](/de/icons/e
...
+If necessary, you can repeat an event at certain intervals. Possible entries are, for example, every four days, every 
+two weeks, every five months or every year.
 
 **Repeat event:** Here you activate the repeat function.
 
 **Interval:** Here you can define the intervals at which the event is repeated.
 
-**Repetitions:** If you enter a value greater than 0 here, the event will not be displayed after the specified number of repetitions.
+**Recurrences:** If you enter a value greater than 0 here, the event will not be displayed after the specified number 
+of repetitions.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:292]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 15]
```diff
@@ -202,79 +249,73 @@ To create a new event, click on the desired archive ![Edit Calendar](/de/icons/e
...
+texts are also evaluated by search engines and are therefore an important tool for onpage optimization.
 
 **Image title:** Here you can enter the title of the image *(title attribute)*.
 
-**Image link address:** If you click on a linked image, you will be redirected to the specified destination page (corresponds to an image link). Please note that for a linked image a lightbox large view is no longer possible.
+**Image link target:** If you click on a linked image, you will be redirected to the specified destination page 
+(corresponds to an image link). Please note that for a linked image a lightbox large view is no longer possible.
 
-**caption:** Here you can enter a caption.
+**Image caption:** Here you can enter a caption.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:280]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 16]
```diff
@@ -166,21 +211,23 @@ To create a new event, click on the desired archive ![Edit Calendar](/de/icons/e
...
 
-**Venue:** Here you can enter a venue for your event.
+**Location name:** Here you can enter the name of the event location.
 
-**address:** Here you can enter the address of the venue.
+**Location address:** Here you can enter the address of the event location.
 
-**Teasertext:** Here you can enter a short summary of the event, which will be displayed e.g. with the module "Event list", followed by a read more link.
+**Event teaser:** Here you can enter a short summary of the event, which will be displayed e.g. with the module 
+"Event list", followed by a read more link.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:227]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 17]
```diff
@@ -166,21 +211,23 @@ To create a new event, click on the desired archive ![Edit Calendar](/de/icons/e
 
 **Meta description**: Here you can enter an individual meta description to override the default page description.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:218]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 18]
```diff
@@ -85,78 +98,110 @@ The URL is:
...
 ### Title and author
 
-**Titles:** Here you can enter the title of the event.
+**Title:** Here you can enter the title of the event.
 
-**Event Alias:** The alias of an event is a unique and meaningful reference that you can use to call it up in your browser.
+**Event Alias:** The alias of an event is a unique and meaningful reference that you can use to call it up in your 
+browser.
 
 **Author:** Here you can change the author of the event.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:173]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 19]
```diff
@@ -85,78 +98,110 @@ The URL is:
...
 
-This section explains how to create an event. Events are always sorted by date, so there are no icons to change the order.
+This section explains how to create an event. Events are always sorted by date, so there are no icons to change the 
+order.
 
 The events consist of the event settings ("Event list") and their contents ("Event reader").
 
-To create a new event, click on the desired archive ![Edit Calendar](/de/icons/edit.svg?classes=icon "Kalender bearbeiten") and then on![Create a new event](/de/icons/new.svg?classes=icon "Ein neues Event erstellen") **New**.
+To create a new event, click on the desired archive ![Edit Calendar](/de/icons/edit.svg?classes=icon "Edit Calendar") 
+and then on![Create a new event](/de/icons/new.svg?classes=icon "Create a new event") **New**.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:164]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 20]
```diff
@@ -85,78 +98,110 @@ The URL is:
...
 
-**Maximum number of posts:** Here you can limit the number of posts in the feed. Usually about 25 posts per feed are sufficient. Most of the time only the first three to five are actually used anyway.
+**Maximum number of items:** Here you can limit the number of posts in the feed. Usually about 25 posts per feed are 
+sufficient. Most of the time only the first three to five are actually used anyway.
 
-**Base URL:** The base URL is especially important in multi-domain operation if you run several websites with one Contao installation. To make sure that the feed links to the correct domain, you can enter it here.
+**Base URL:** The base URL is especially important in multi-domain operation if you run several websites with one 
+Contao installation. To make sure that the feed links to the correct domain, you can enter it here.
 
 **Feed description:** Here you can enter a description of the feed.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:154]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 21]
```diff
@@ -85,78 +98,110 @@ The URL is:
...
 
-**Feed Alias:** The alias of a feed is used as the file name.
+**Feed alias:** The alias of a feed is used as the file name.
 
 **Feed language:** Here you can enter the language of the [feed](http://www.rssboard.org/rss-language-codes#table).
 
-### Calendar
+### Calendars
 
 **Calendars:** Here you define which calendars are included in the feed.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:139]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 22]
```diff
@@ -85,78 +98,110 @@ The URL is:
...
+**RSS feeds** and then click on ![Create a new feed](/de/icons/new.svg?classes=icon "Create a new feed") **New**.
 
 ### Title and language
 
 **Title:** The title is output as a feed title in the XML file.
 
-**Feed Alias:** The alias of a feed is used as the file name.
+**Feed alias:** The alias of a feed is used as the file name.
 
 **Feed language:** Here you can enter the language of the [feed](http://www.rssboard.org/rss-language-codes#table).
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:135]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 23]
```diff
@@ -85,78 +98,110 @@ The URL is:
...
+            <guid>https://www.example.com/event/alias-of-the-appointment.html</guid>
         </item>
         …
      </channel>
 </rss>
 ```
 
-To create a new feed click on ![Manage RSS feeds](/de/icons/rss.svg?classes=icon "RSS-Feeds verwalten") **RSS feeds** and then click on ![Create a new feed](/de/icons/new.svg?classes=icon "Einen neuen Feed erstellen") **New**.
+To create a new feed click on ![Manage calendar feeds](/de/icons/rss.svg?classes=icon "Manage calendar feeds") 
+**RSS feeds** and then click on ![Create a new feed](/de/icons/new.svg?classes=icon "Create a new feed") **New**.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:126]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 24]
```diff
@@ -63,15 +70,21 @@ You already know the Contao comment function from the "News/Blog" extension or t
...
 
-**Login required for commenting:** If you select this option, only logged in members can add comments. However, comments already submitted will still be visible to all visitors of the website.
+**Require login to comment:** If you select this option, only logged in members can add comments. However, comments 
+already submitted will still be visible to all visitors of the website.
 
-**Disable spam protection:** By default, visitors are required to answer a security question when creating comments to prevent the commenting feature from being misused for spam purposes. However, if you want to allow only logged in members to comment, you can disable the security question here. Since Contao 4.4, this question is only "displayed" to spambots.
+**Disable spam protection:** By default, visitors are required to answer a security question when creating comments to 
+prevent the commenting feature from being misused for spam purposes. However, if you want to allow only logged in 
+members to comment, you can disable the security question here. Since Contao 4.4, this question is only "displayed" to 
+spambots.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:83]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 25]
```diff
@@ -1,52 +1,59 @@
...
-Just like content elements, calendars can also be protected. The events of the calendar are then only shown to registered members.
+Just like content elements, calendars can also be protected. The events of the calendar are then only shown to 
+registered members.
 
-**Protect archive:** Here you activate the access protection.
+**Protect calendar:** Here you activate the access protection.
 
-**Allowed member groups:** Here you define which member groups should have access to the calendar after logging in to the frontend.
+**Allowed member groups:** Here you define which member groups should have access to the calendar after logging in to 
+the frontend.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:39]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 26]
```diff
@@ -1,52 +1,59 @@
...
+To create a new calendar, click ![Create a new calendar](/de/icons/new.svg?classes=icon "Create a new calendar") **New**.
 
-### Title and forwarding
+### Title and redirect page
 
 **Title:** The title of a calendar is used in the backend overview.
 
-**Forwarding page**: Here you can define to which page a visitor is forwarded when clicking on an event. This target page should contain the module "Eventleser" to display the complete article.
+**Redirect page***: Here you define which page a visitor is redirected to when clicking on an event. This landing page 
+should contain the Event Reader module to display the full post.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:27]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 27]
```diff
@@ -1,52 +1,59 @@
...
+You can create multiple calendars there, which in turn contain the individual appointments or events. By using multiple 
+calendars, a categorization of the entries is possible.
 
 ## Calendar
 
-Archives are used to group and/or categorize calendars. Each archive can refer to a specific language or topic.
+Archives are used to group and/or categorize calendars. Each archive can be related to a specific language or topic.
 
-To create a new calendar, click ![Create a new calendar](/de/icons/new.svg?classes=icon "Einen neuen Kalender erstellen") **New**.
+To create a new calendar, click ![Create a new calendar](/de/icons/new.svg?classes=icon "Create a new calendar") **New**.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:20]:
```suggestion


```

File: docs/manual/core-extensions/calendar/calendar-management.en.md [hunk 28]
```diff
@@ -1,52 +1,59 @@
...
 ---
 
-{{% notice warning %}}
-This article is machine translated.
-{{% /notice %}}
-
-The event management is a separate module in the backend called "Events", which you can find in the group "Contents". There you can create several calendars, which contain the individual appointments or events. By using several calendars, a categorization of the entries is possible.
+The event management is a separate module in the backend called "Events", which you can find in the "Content" group. 
+You can create multiple calendars there, which in turn contain the individual appointments or events. By using multiple 
+calendars, a categorization of the entries is possible.
```
Comment (xchs) [inline docs/manual/core-extensions/calendar/calendar-management.en.md:13]:
```suggestion


```

**Review by xchs — COMMENTED** [review]
👍🏻

**Comment by netzarbeiter** [comment]
Thank you, Hannes

**Pull request merged**